src/Tizen.NUI/src/public/BaseComponents/ViewAccessibilityV2.cs

```diff
@@ -0,0 +1,328 @@
+/*
+ * Copyright(c) 2025 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+using System;
+using System.ComponentModel;
+using System.Runtime.InteropServices;
+using Tizen.NUI;
+
+namespace Tizen.NUI.BaseComponents
+{
+    /// <summary>
+    /// AccessibilityStateV2 is an enumeration that represents the states of the view to send to accessibility service.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public enum AccessibilityStateV2
+    {
+        /// <summary>
+        /// Indicates whether the view is enabled or not.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        Enabled = 0,
+        /// <summary>
+        /// Indicates whether a selectable element is currently selected or not.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        Selected,
+        /// <summary>
+        /// Indicates the state of a checkable element.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        Checked,
+        /// <summary>
+        /// Indicates whether an element is currently busy or not.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        Busy,
+        /// <summary>
+        /// Indicates whether an expandable element is currently expanded or collapsed.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        Expanded
+    }
+
+    /// <summary>
+    /// The AccessibilityStatesV2 structure represents a set of states of the view to communicate to accessibility service.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public struct AccessibilityStatesV2
+    {
+        const uint _on = 1;
+        private uint _bitMask;
```

Comment:
Question.
Which convention rule is right for field?
using "_" or lower charactor?

Reply:
I just follow the Dotnet coding style below :
https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/identifier-names#camel-case

Reply:
https://confluence.sec.samsung.net/display/GFX/NUI+Ground+Rule
It is defined NUI coding rule. (Alhtough most NUI code are not following this)

> C# coding style follows dotnet runtime guidelines
https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md
In addition to rule3, use "c_" prefix for const (or readonly) field
c_somePrivateConstFieldName

Reply:
I think I also need to add `c_` prefix for `_on` variable, don't I?